lab_solutions/lab02_06_build_wikipedia_corpus.py

```diff
@@ -0,0 +1,44 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright © 2020 Attila Nagy <attila.nagy234@gmail.com>
+#
+# Distributed under terms of the MIT license.
+
+import wikipedia
+
+
+def build_wikipedia_corpus():
+    wikipedia.set_lang('hu')
+    cities_to_search = ['Budapest', 'Debrecen', 'Miskolc', 'Nyíregyháza', 'Pécs', 'Szeged']
+    corpus_length = 0
+    corpus = []
+    for city in cities_to_search:
+        city_page = wikipedia.page(city)
+        if corpus_length <= 100000:
+            corpus.append(city_page.content)
+            corpus_length += len(city_page.content)
+        else:
+            return corpus
+
+    raise Exception('Could not exceed the desired 100000 corpus length. Please provide more pages to search.')
```

Comment:
Line too long (>79). There are other long lines as well. Set your editor up to warn you about these.